tests/playbooks/roles/install-devstack/templates/local.conf.j2

```diff
@@ -0,0 +1,116 @@
+[[local|localrc]]
+RECLONE=False
+HOST_IP={{ local_ip_address }}
+DEST=/opt/stack
+DATA_DIR=${DEST}/data
+USE_PYTHON3=True
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=False
+LOGDAYS=1
+SERVICE_TIMEOUT=300
+
+DATABASE_PASSWORD=password
+ADMIN_PASSWORD=password
+SERVICE_PASSWORD=password
+SERVICE_TOKEN=password
+RABBIT_PASSWORD=password
+
+TARGET_BRANCH={{ branch }}
+
+ENABLED_SERVICES={{ enable_services | join(',') }}
```

Comment:
Can we also define `enable_services ` in `defaults/main.yml`?

Reply:
Added in https://github.com/kubernetes/cloud-provider-openstack/pull/1651/commits/624942951770345fa30b065ef43e2f6eeaf440da